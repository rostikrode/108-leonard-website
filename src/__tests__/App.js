// import 'babel-polyfill';
// import React from 'react';
// import ReactDOM from 'react-dom';
// import { mount } from 'enzyme';
// import App from '../components/App';
// import { MemoryRouter } from 'react-router-dom';
//
// window.gtag = window.gtag || function() {
//   return true;
// };
//
// it('renders without crashing', () => {
//   const tree = mount(
//     <MemoryRouter><App  /></MemoryRouter>
//   );
//   expect(tree.html()).toMatchSnapshot()
// });
