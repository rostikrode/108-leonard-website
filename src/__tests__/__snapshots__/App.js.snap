// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div class=\\"App\\"><header class=\\"header\\"><button class=\\"ham-nav-button \\"><div class=\\"ham-nav \\"><span></span><span></span><span></span><span></span></div></button><div class=\\"mobile-header\\"><div class=\\"navigation-titles\\"><h4 class=\\"title\\"><img src=\\"108_leonard_text.svg\\" alt=\\"108 Leonard text logo\\"></h4><h3 class=\\"come-in sub-title sans-light-bold\\"><!-- react-text: 14 --><!-- /react-text --><!-- react-text: 15 --> <!-- /react-text --><!-- react-text: 16 --><!-- /react-text --></h3></div></div><div class=\\"app-header \\"><a class=\\"active\\" aria-current=\\"true\\" data-type=\\"\\" id=\\"home-0\\" href=\\"/\\"><img src=\\"logo.svg\\" class=\\"app-logo\\" alt=\\"logo\\"></a><nav><ul class=\\"nav-list\\"><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"carousel\\" id=\\"nav-anchor-0\\" href=\\"/building/\\">Building</a><ul class=\\"nav-subnav\\"><li role=\\"button\\" data-id=\\"Property\\" class=\\"nav-subnav-item sans-light\\">Property</li><li role=\\"button\\" data-id=\\"Entrance\\" class=\\"nav-subnav-item sans-light\\">Entrance</li></ul></li><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"carousel\\" id=\\"nav-anchor-1\\" href=\\"/residences/\\">Residences</a><ul class=\\"nav-subnav\\"><li role=\\"button\\" data-id=\\"Interiors\\" class=\\"nav-subnav-item sans-light\\">Interiors</li><li role=\\"button\\" data-id=\\"Kitchens\\" class=\\"nav-subnav-item sans-light\\">Kitchens</li><li role=\\"button\\" data-id=\\"Bathrooms\\" class=\\"nav-subnav-item sans-light\\">Bathrooms</li></ul></li><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"\\" id=\\"nav-anchor-2\\" href=\\"/penthouses/\\">Penthouses</a><!-- react-text: 35 --><!-- /react-text --></li><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"\\" id=\\"nav-anchor-3\\" href=\\"/availability/\\">Availability</a><!-- react-text: 38 --><!-- /react-text --></li><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"carousel\\" id=\\"nav-anchor-4\\" href=\\"/amenities/\\">Amenities</a><ul class=\\"nav-subnav\\"><li role=\\"button\\" data-id=\\"Entertaining\\" class=\\"nav-subnav-item sans-light\\">Entertaining</li><li role=\\"button\\" data-id=\\"Wellness\\" class=\\"nav-subnav-item sans-light\\">Wellness</li><li role=\\"button\\" data-id=\\"Outdoor\\" class=\\"nav-subnav-item sans-light\\">Outdoor</li></ul></li><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"carousel\\" id=\\"nav-anchor-5\\" href=\\"/tribeca/\\">Tribeca</a><ul class=\\"nav-subnav\\"><li role=\\"button\\" data-id=\\"Neighborhood\\" class=\\"nav-subnav-item sans-light\\">Neighborhood</li><li role=\\"button\\" data-id=\\"Map\\" class=\\"nav-subnav-item sans-light\\">Map</li></ul></li><li class=\\"nav-anchor-wrapper\\"><a class=\\"nav-anchor sans\\" aria-current=\\"false\\" data-type=\\"\\" id=\\"nav-anchor-6\\" href=\\"/contact/\\">Contact</a><!-- react-text: 52 --><!-- /react-text --></li></ul></nav><div class=\\"footer\\"><div class=\\"footer-pages\\"><a class=\\"nav-anchor sans-medium\\" aria-current=\\"false\\" data-type=\\"carousel\\" id=\\"team-0\\" href=\\"/team/\\">Team</a><a class=\\"nav-anchor sans-medium\\" aria-current=\\"false\\" data-type=\\"\\" id=\\"press-0\\" href=\\"/press/\\">Press</a><a class=\\"nav-anchor sans-medium\\" aria-current=\\"false\\" data-type=\\"\\" id=\\"legal-0\\" href=\\"/legal/\\">Legal</a></div><div class=\\"social-icons\\"><a rel=\\"noopener noreferrer\\" href=\\"https://www.instagram.com/108_leonard/\\" target=\\"_blank\\"><img class=\\"insta\\" src=\\"insta.svg\\" alt=\\"Instagram @108_leonard\\"></a><a rel=\\"noopener noreferrer\\" href=\\"https://www.facebook.com/108leonardst/\\" target=\\"_blank\\"><img class=\\"facebook\\" src=\\"facebook.svg\\" alt=\\"Facebook at 108leonardst\\"></a></div></div></div></header><main style=\\"padding-top: 0px;\\"><div class=\\"slider-parent home-slider-parent\\"><div class=\\"slick-initialized slick-slider home-slider\\"><div class=\\"slick-list\\"><div class=\\"slick-track\\" style=\\"opacity: 1; transform: translate3d(0px, 0px, 0px);\\"><div data-section=\\"Home\\" class=\\"slick-slide slick-active slick-section\\" data-index=\\"0\\" tabindex=\\"-1\\" style=\\"outline: none; position: relative; left: 0px; opacity: 1; width: 0px;\\"><div class=\\"inner\\"><!-- react-text: 70 --><!-- /react-text --><div class=\\"image-wrapper\\"><div class=\\"loading-wrapper \\"><i class=\\"loading\\"></i></div></div><p class=\\"home-caption serif-bold\\">Grand Entrance on Leonard Street</p></div></div><div data-section=\\"Home\\" class=\\"slick-slide slick-section\\" data-index=\\"1\\" tabindex=\\"-1\\" style=\\"outline: none; position: relative; left: 0px; opacity: 0; width: 0px;\\"><div class=\\"inner\\"><!-- react-text: 77 --><!-- /react-text --><div class=\\"image-wrapper\\"><div class=\\"loading-wrapper \\"><i class=\\"loading\\"></i></div></div><p class=\\"home-caption serif-bold\\">Motor Reception</p></div></div><div data-section=\\"Home\\" class=\\"slick-slide slick-section\\" data-index=\\"2\\" tabindex=\\"-1\\" style=\\"outline: none; position: relative; left: 0px; opacity: 0; width: 0px;\\"><div class=\\"inner\\"><!-- react-text: 84 --><!-- /react-text --><div class=\\"image-wrapper\\"><div class=\\"loading-wrapper \\"><i class=\\"loading\\"></i></div></div><p class=\\"home-caption serif-bold\\">A Modern Renaissance</p></div></div><div data-section=\\"Home\\" class=\\"slick-slide slick-section\\" data-index=\\"3\\" tabindex=\\"-1\\" style=\\"outline: none; position: relative; left: 0px; opacity: 0; width: 0px;\\"><div class=\\"inner\\"><!-- react-text: 91 --><!-- /react-text --><div class=\\"image-wrapper\\"><div class=\\"loading-wrapper \\"><i class=\\"loading\\"></i></div></div><p class=\\"home-caption serif-bold\\">Terrace Penthouse</p></div></div></div></div></div><a class=\\"nav-anchor serif-bold next-button\\" aria-current=\\"false\\" href=\\"/building/\\"><span><!-- react-text: 98 -->Go to <!-- /react-text --><!-- react-text: 99 -->Building<!-- /react-text --></span><img src=\\"next_arrow.svg\\" alt=\\"arrow to take you to next carousel page\\"></a></div></main></div>"`;
